Resources/config/helper.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<container xmlns="http://symfony.com/schema/dic/services"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <service id="sonata.article.helper.fragment" class="Sonata\ArticleBundle\Helper\FragmentHelper">
+            <argument type="service" id="templating"/>
+        </service>
+    </services>
+</container>
```

Comment:
Please configure your IDE to always add a newline, it will make this red pictogram disappear.